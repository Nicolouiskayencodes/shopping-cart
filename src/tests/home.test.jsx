import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Home from "../components/home";

describe("Home component", () => {
  it("renders cart", ()=>{
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  })
});