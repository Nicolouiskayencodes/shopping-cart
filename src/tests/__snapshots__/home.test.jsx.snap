// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home component > renders cart 1`] = `
<div>
  <div
    class="_container_3f2dd1"
  >
    <img
      alt=""
      class="_img_3f2dd1"
      src="/src/assets/checkout.jpg"
    />
    <div
      class="_right_3f2dd1"
    >
      <p
        class="_info_3f2dd1"
      >
        Welcome to the InterPorium! We are a sell a large assortment of high quality and affordable goods. We source our products from community oriented and environmentally friendly small businesses worldwide! We are dedicated to positive customer experiences and linking buyers and sellers across the globe. If you are interested in getting in contact, email us at interporium@fakebusinessinquiries.net. We hope you like what you see!
      </p>
      <p
        class="_extra_3f2dd1"
      >
        Photo by 
        <a
          href="https://unsplash.com/@blakewisz?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash"
        >
          Blake Wisz
        </a>
         on 
        <a
          href="https://unsplash.com/photos/person-holding-smartphone-beside-tablet-computer-Xn5FbEM9564?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash"
        >
          Unsplash
        </a>
      </p>
      <div
        class="_extra_3f2dd1"
      >
        InterPoriumâ„¢ nicolouiskayencodes 2024
      </div>
    </div>
  </div>
</div>
`;
